Dockerfile

```diff
@@ -18,6 +18,7 @@ RUN apt-get update && apt-get install --no-install-recommends -y \
     pkgconf \
     python3-dev \
     python3-gi \
+    python3-libdiscid \
```

Comment:
`python3-libdiscid` is the package name in the repositories of Debian & Ubuntu of the python module called `libdiscid` ([git repository](https://github.com/sebastinas/python-libdiscid)) which is different from the `discid` python module ([git repository](https://github.com/JonnyJD/python-discid), not packaged in Debian/Ubuntu).
Elsewhere in this pull request it seems you're using `discid` so I would suggest sticking with that one (`discid`).

Reply:
You need to add `libdiscid0` instead of `python3-libdiscid` here.

Reply:
Then replace the following line:

```bash
    && pip3 --no-cache-dir install pycdio==2.1.0
```
With this:
```bash
    && pip3 --no-cache-dir install pycdio==2.1.0 discid
```
To install the `discid` module using pip.

Reply:
I have already replaced `python3-libdiscid` with `libdiscid0` in travis but forgot the Dockerfile :DD.
Thank you!